.github/workflows/call-build-linux-arm-packages.yml

```
@@ -52,7 +52,15 @@ jobs:
         run: sudo apt-get update && sudo apt-get install -y build-essential libssl-dev libsystemd-dev
 
       - name: Make Valkey
-        run: make -C src all BUILD_TLS=yes USE_SYSTEMD=yes USE_FAST_FLOAT=yes
+        run: |
+          MAJOR_MINOR=$(echo "${{ inputs.version }}" | cut -d. -f1,2)
+          
+          if [[ $(echo "$MAJOR_MINOR >= 8.0" | bc -l) -eq 1 ]]; then
+            echo "Version $MAJOR_MINOR is 8.0 or higher, using USE_FAST_FLOAT"
```

```suggestion
          if [[ $(echo "$MAJOR_MINOR >= 8.1" | bc -l) -eq 1 ]]; then
            echo "Version $MAJOR_MINOR is 8.1 or higher, using USE_FAST_FLOAT"
```
Also add unstable version to use fast float